docs/develop/smart-contracts/custom-improvement-proposals.md

```diff
@@ -0,0 +1,114 @@
+# Custom Improvement Proposals
+
+With the release [v19.0.0](https://github.com/evmos/evmos/releases/tag/v19.0.0)
+a new feature called custom improvement proposals has been introduced in the
+evmOS framework. Custom improvement proposals allow protocol developers to
+modify the behavior of the EVM opcodes to tailor their functionalities to the
+specific needs.
+
+Improvement proposals are a way to introduce new standards, aimed to improve
+protocol functionalities. Changes proposed in this way can affect any aspect
+of the protocol but are mainly used to customize the behavior of
+smart contract execution.
```

Comment:
I think this can actually be removed altogether, I think it's sufficiently explained in the paragraph above
```suggestion
```